views/term_table.erb

```diff
@@ -198,7 +198,7 @@
       <div class="page-section page-section--grey source-credits">
         <div class="container">
           <% if @page.data_sources %>
-            <p>Main Source<% if @page.data_sources.size > 1 %>s<% end %>: <%= @page.data_sources.map { |url| %Q(<a href="#{url}">#{url}</a>) }.join(", ") %></p>
+            <p>Main Source<% if @page.data_sources.size > 1 %>s<% end %>: <%= @page.data_sources.map { |url| %Q(<a href="#{escape_uri(url)}">#{url}</a>) }.join(", ") %></p>
```

Comment:
This approach concerns me. If the problem is that EP is giving us invalid URLs, then we should fix that upstream, so that everyone consuming the data doesn't need to do things like this. 

Whichever approach we take, however, this feels like something that deserves a PR of its own, properly explaining the problem and solution etc., rather than cramming more into _this_ PR.